packages/engine/src/experiment/controller/run.rs

```diff
@@ -346,3 +349,38 @@ fn worker_pool_exit_logic(
 
     false
 }
+
+#[derive(PartialEq, Eq, Clone)]
+pub enum EngineExitStatus {
```

Comment:
As I said in another comment chain, I don't see this as providing any value right now. It's literally used here:

```rust
                match exit_status {
                    EngineExitStatus::Success => tracing::error!(
                        experiment = %experiment_id,
                        "Removed file {path:?} that should've been cleanup by the engine."
                    ),
                    EngineExitStatus::Error => tracing::warn!(
                        experiment = %experiment_id,
                        "Removed file {path:?} that should've been cleanup by the engine."
                    ),
                };
```

Reply:
Removed `EngineExitStatus` in https://github.com/hashintel/hash/pull/494/commits/ce7c3a4f39d8e261bc0ff35dbbd01471becbc471